### ISSUE TITLE
Debug angular 

### ISSUE THREAD

**Issue by dave900** [maximegris/angular-electron#355]
Hello,

Is it possible to debug Main Process and Renderer Process (Angular in this case) code from VSCode while having hot-reload feature at the same time?
What would the launch.json look like?

Thanks for the help!

**Comment by nikhilnxvverma1** [comment]
This is mostly covered in #84

**Comment by dave900** [comment]
Thanks for the link!! I could get it to work for the Main Process.
For Angular + Renderer process, I could make it work but without hot reload.
--> Any way to have hot reload work as well as debuging in VSCode?

Here are both of my debug configs:
```
{
	"type": "node",
	"request": "launch",
	"name": "Electron Main",
	"program": "${workspaceFolder}/main.ts",
	"outFiles": [
		"${workspaceRoot}/main.js",
	],
	"cwd": "${workspaceRoot}",
	"runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron",
	"windows": {
		"runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
	},
	"args": ["--serve"],
	"runtimeArgs": [
		".",
	],
	"outputCapture": "std",
	"preLaunchTask": "npm: electron:serve-tsc",
},
{
	"name": "Electron: Renderer",
	"type": "chrome",
	"request": "launch",
	"cwd": "${workspaceRoot}",
	"runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron",
	"windows": {
		  "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
	},
	"runtimeArgs": [
		".",
		"--remote-debugging-port=9222"
	],
	"sourceMapPathOverrides": {
		"webpack:\\*": "${workspaceRoot}/*"
	},
	"sourceMaps": true,
	"timeout": 30000,
	"preLaunchTask": "npm: build:dev",
	"webRoot": "${workspaceRoot}"
 }
```

**Comment by nikhilnxvverma1** [comment]
I mildly experimented with __chrome__ type launch processes in VSCode but to little success. What I do, I simply trigger off `ng serve` or equivalent in terminal and debug my application directly inside chrome. You easily get sourcemap support and you can continue to edit your application in VSCode(UI Process part) and the hot reload will kick in. While I am not sure, I don't think `build:dev` watches for changes like `ng serve`.

So in a nutshell :
Debug Main process in VSCode + Debug UI Process in chrome(with hot reload)

**Comment by dave900** [comment]
Thanks for the update, so I'll keep debugging in Chrome and not directly within VSCode :)

**Comment by ouss4m4** [comment]
you ever found a solution to that ? i can't debug the angular app in vscode
and if i do inside chrome, it shows the JS files not the TS
